src/main/java/seedu/address/model/person/Person.java

```diff
@@ -44,7 +46,8 @@ public Person(Name name, Phone phone, Email email, Address address, PostalCode p
      * Creates a copy of the given ReadOnlyPerson.
      */
     public Person(ReadOnlyPerson source) {
-        this(source.getName(), source.getPhone(), source.getEmail(), source.getAddress(), source.getPostalCode(),
+        this(source.getName(), source.getPhone(), source.getEmail(), source.getAddress(), source.getFormClass(),
```

Comment:
please standardize the positioning of attributes. In line 40, position is postalCode followed by formClass.

Reply:
The positioning of the assignment doesn't matter but okay, I'll change it. Just point out the other instances if any, and I'll change them too.